.gitignore

```diff
@@ -7,4 +7,4 @@ lightning-c-bindings/a.out
 **/*.rs.bk
 Cargo.lock
 .idea
-
+lightning/target
```

Comment:
Is this needed if you open the editor at the root of the repository rather than the `lightning` directory?

Reply:
Yup. `lightning/target` was not being ignored even when open from the project root.